docs/cli-options.md

```diff
@@ -45,6 +45,13 @@ as well as the text format for WebAssembly (`*.wat`):
 $ wasmtime foo.wat
 ```
 
+The `run` command accepts an optional `invoke` argument which is the name of
+an exported function of the module to run.
+
+```sh
+$ wasmtime foo.wasm --invoke initialize
```

Comment:
I know that we talk about how "run" is optional just above, but it reads strange to me to talk about the `run` command but then not have `run` in the command.

```suggestion
$ wasmtime run foo.wasm --invoke initialize
```